nylas/client/client.py

```diff
@@ -270,6 +295,19 @@ def availability(
             end_time = timestamp_from_dt(end_at)
         else:
             end_time = end_at
+
+        if open_hours is not None:
+            free_busy_emails = (
+                [fb["email"] for fb in free_busy] if free_busy is not None else []
+            )
+            for email in open_hours["emails"]:
```

Comment:
`open_hours` as passed into the availability endpoint should be a list of objects "open_hours" objects as defined above

Reply:
Thanks, fixed!